src/mmap.cpp

```diff
@@ -111,27 +109,24 @@ mmap_reads(off_t partial_file_size,
       else
       {
         // count the number of strings in the file section
-        for ( int64_t i = reads_offset_f; i < partial_file_size-reads_offset_e-2; i++ ) if ( raw[i] == '>' ) strs++;
-
+        for ( int64_t i = reads_offset_f; i < partial_file_size-reads_offset_e-2; i++ ) if ( raw[i] == '>' )
```

Comment:
would it be possible to break the if-statement out onto a separate line? this is a bit complex to look at.

also, what is the significance of the `-2`?